src/index.ts

```diff
@@ -39,8 +39,8 @@ export * from './locale';
 export { fakerEN as faker } from './locale';
 export * from './locales';
 export * as allLocales from './locales';
-export { Aircraft, AircraftType } from './modules/airline';
-export type { AirlineModule } from './modules/airline';
+export { Aircraft } from './modules/airline';
+export type { AircraftType, AirlineModule } from './modules/airline';
```

Comment:
Should we sort this alphabetically?

```suggestion
export type { AirlineModule, AircraftType } from './modules/airline';
```